src/multi_vector_simulator/D1_model_components.py

```diff
@@ -577,6 +579,8 @@ def storage_fix(model, dict_asset, **kwargs):
         },  # maximum discharge possible in one timestep
         loss_rate=1
         - dict_asset[STORAGE_CAPACITY][EFFICIENCY][VALUE],  # from timestep to timestep
+        fixed_losses_absolute=dict_asset[STORAGE_CAPACITY][THERM_LOSSES_ABS][VALUE],
+        fixed_losses_relative=dict_asset[STORAGE_CAPACITY][THERM_LOSSES_REL][VALUE],
```

Comment:
Unfortunately with this I can only pass one numeric value if my storage_xx.csv file looks like this:

```
...
fixed_thermal_losses_relative,factor,0.002,,
fixed_thermal_losses_absolute,kWh,0.0003,,
```

But how can I pass a scalar out of this storage_xx.csv:

```
...
fixed_thermal_losses_relative,factor,"{'file_name': 'None', 'header': 'no_unit', 'unit': ''}",,
fixed_thermal_losses_absolute,kWh,"{'file_name': 'None', 'header': 'no_unit', 'unit': ''}",,
```

?

Reply:
Solved with commit https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/a76b9115758b192db016eafe5f9033ff9a009d47